docs/data/sql_functions.yml

```diff
@@ -228,6 +228,13 @@ arithmetic:
   - sql: UUID()
     table: uuid()
     description: Returns an UUID (Universally Unique Identifier) string (e.g., "3d3c68f7-f608-473f-b60c-b0c44ad4cc4e") according to RFC 4122 type 4 (pseudo randomly generated) UUID. The UUID is generated using a cryptographically strong pseudo random number generator.
+  - sql: |
+      CONV(numeric, numeric1, numeric2)
+      CONV(string, numeric1, numeric2)
```

Comment:
What happens if input will contain number `Long.MAX_VALUE + 5` ?
Doc says nothing about it however there are some checks in the code for `Long.MAX_VALUE`

Reply:
the main code from Hive， you can see this comment: org.apache.hadoop.hive.ql.udf.UDFConv.java. Just modify conv function
 * return type to prevent codec overhead @snuyanzin